src/SonarScanner.MSBuild.TFS.Classic/BinaryToXmlCoverageReportConverter.cs

```diff
@@ -75,33 +41,14 @@ public BinaryToXmlCoverageReportConverter(ILogger logger, AnalysisConfig config)
         public BinaryToXmlCoverageReportConverter(IVisualStudioSetupConfigurationFactory setupConfigurationFactory,
```

Comment:
The `IVisualStudioSetupConfigurationFactory` or parts of can also be removed in a follow-up PR.